alonzo/impl/src/Cardano/Ledger/Alonzo/PParams.hs

```diff
@@ -151,7 +152,9 @@ data PParams' f era = PParams
     -- | Max total script execution resources units allowed per tx
     _maxTxExUnits :: !(HKD f ExUnits),
     -- | Max total script execution resources units allowed per block
-    _maxBlockExUnits :: !(HKD f ExUnits)
+    _maxBlockExUnits :: !(HKD f ExUnits),
+    -- | Max size of a Value in an output
+    _maxValSize :: !(HKD f Int64)
```

Comment:
I think a non-negative type would make more sense.

Reply:
Natural it will be